README.md

```diff
@@ -41,8 +41,8 @@ NuoDB Insights is a visual monitor tool that aids NuoDB practitioners in monitor
 |-----------|--------------------------------------------------------|
 | [NuoDB](https://nuodb.com/) | A distributed SQL database. 4.0 or newer |
 | [NuoDB Collector](https://github.com/nuodb/nuodb-collector)   | The stats collector daemon. 1.1.0 or newer |
-| [InfluxDB](https://portal.influxdata.com/downloads/)    | Time-series database. Only version 1.X is supported. NuoDB recommends version 1.8 or later.  |
-| [Grafana](https://grafana.com/grafana/download)    | Dashboards visualization. Only version 7.X is supported. NuoDB recommends version 7.5.4 or later. |
```

Comment:
Once we test the new Grafana version, I think that we should pin the support to 9.x.